autopush/web/webpush.py

```diff
@@ -204,6 +204,48 @@ def reject_encryption_key(self, data, original_data):
             )
 
 
+class WebPushCrypto06HeaderSchema(Schema):
+    """Validate webpush Message Encryption
+
+    Uses draft-ietf-webpush-encryption-06 rules for validation
+
+    """
+    content_encoding = fields.String(
+        required=True,
+        load_from="content-encoding",
+        validate=OneOf(["aes128gcm"])
```

Comment:
nitpick of something else copied from the other Schemas: this could be Equals(...) instead of OneOf([...])